docker/install.R

```diff
@@ -1,3 +1,12 @@
+# 2020-10-06 (see https://packagemanager.rstudio.com/client/#/repos/1/overview)
+snapshot <- "https://packagemanager.rstudio.com/all/__linux__/focal/342"
+
+# Build an image with R packages from a specific date `snapshot`:
+# docker build \
+#     --build-arg BUILD_DATE=yyyy-mm-dd \
```

Comment:
Non-blocking: Do we know what CRAN's retention policy  for old package binaries is?